CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+### [2.73.0]
+Clean up of many old components and replaced Stream SDK with Service SDK 14
+> By moving to Service SDK, pipelines can now be configured like any other Backbase service using the Service SDK
+> 
+> **Migrate your CICD pipelines to the Service SDK standards**
+
+
+### Removed
+- Old Legal Entity Open API definitions
+- Stream Transactions Open API Spec
+- Removed Spring Cloud Data Flow components as nobody uses it
+  - Stream Cursor Source
+  - Legal Entity Sink
+  - Product Sink
+  - Transactions Sink
+- Removed Stream SDK Starters
+  - `stream-aio-starter-parent` (replaced by `service-sdk-core-starter`)
+  - `stream-batch-starter-parent` (replaced by `service-sdk-starter-core` + `spring-boot-starter-batch`)
+  - `stream-generated-client-starter-parent`
+  - `stream-processor-starter-parent`
+  - `stream-sdk-starter-core`(replaced by `service-sdk-starter-core`)
+  - `stream-sink-starter-parent`
+  - `stream-source-starter-parent`
+- Removed `stream-dbs-web-client` (replaced by `service-sdk-web-client`)
```

Comment:
we need to inform the breaking change here: where the `dbs` client now becomes the `bb` client, so whoever adopts this needs to update the configuration accordingly. Ideally it would be great to find a way to keep it retrocompatible

Reply:
Actually. I did add these instructions, but it actually just worked fine with the dbs setting as well!